src/main/java/no/nav/foreldrepenger/oversikt/arbeid/ArbeidRest.java

```diff
@@ -0,0 +1,126 @@
+import no.nav.foreldrepenger.oversikt.saker.AnnenPartSakTjeneste;
+import no.nav.foreldrepenger.oversikt.saker.BrukerIkkeFunnetIPdlException;
+import no.nav.foreldrepenger.oversikt.saker.InnloggetBruker;
+import no.nav.foreldrepenger.oversikt.saker.PersonOppslagSystem;
+import no.nav.foreldrepenger.oversikt.tilgangskontroll.TilgangKontrollTjeneste;
+import no.nav.fpsak.tidsserie.LocalDateInterval;
+
+@Path("/arbeid")
+@ApplicationScoped
+@Transactional
+public class ArbeidRest {
+
+    private static final Logger LOG = LoggerFactory.getLogger(ArbeidRest.class);
+
+    private AnnenPartSakTjeneste annenPartSakTjeneste;
+    private TilgangKontrollTjeneste tilgangkontroll;
+    private InnloggetBruker innloggetBruker;
+    private PersonOppslagSystem personOppslagSystem;
+    private MineArbeidsforholdTjeneste mineArbeidsforholdTjeneste;
+    private AktivitetskravArbeidDokumentasjonsKravTjeneste aktivitetskravArbeidDokumentasjonsKravTjeneste;
+
+    @Inject
+    public ArbeidRest(AnnenPartSakTjeneste annenPartSakTjeneste, TilgangKontrollTjeneste tilgangkontroll, InnloggetBruker innloggetBruker,
+                      PersonOppslagSystem personOppslagSystem, MineArbeidsforholdTjeneste mineArbeidsforholdTjeneste,
+                      AktivitetskravArbeidDokumentasjonsKravTjeneste aktivitetskravArbeidDokumentasjonsKravTjeneste) {
+        this.annenPartSakTjeneste = annenPartSakTjeneste;
+        this.tilgangkontroll = tilgangkontroll;
+        this.innloggetBruker = innloggetBruker;
+        this.personOppslagSystem = personOppslagSystem;
+        this.mineArbeidsforholdTjeneste = mineArbeidsforholdTjeneste;
+        this.aktivitetskravArbeidDokumentasjonsKravTjeneste = aktivitetskravArbeidDokumentasjonsKravTjeneste;
+    }
+
+    ArbeidRest() {
+    }
+
+    @Path("/mittArbeid")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public List<EksternArbeidsforhold> hentMittArbeid() {
+        tilgangkontroll.sjekkAtKallErFraBorger();
+        tilgangkontroll.tilgangssjekkMyndighetsalder();
+
+        return mineArbeidsforholdTjeneste.brukersArbeidsforhold(innloggetBruker.fødselsnummer());
+    }
+
+    @Path("/mineFrilansoppdrag")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public List<EksternArbeidsforhold> hentMineFrilansoppdrag() {
+        tilgangkontroll.sjekkAtKallErFraBorger();
+        tilgangkontroll.tilgangssjekkMyndighetsalder();
+
+        return mineArbeidsforholdTjeneste.brukersFrilansoppdragSisteSeksMåneder(innloggetBruker.fødselsnummer());
+    }
+
+    @Path("/morDokuentasjon")
+    @POST
+    @Produces(MediaType.APPLICATION_JSON)
+    public boolean trengerDokumentereMorsArbeid(@Valid @NotNull ArbeidRest.MorArbeidRequest request) {
+        tilgangkontroll.sjekkAtKallErFraBorger();
+        tilgangkontroll.tilgangssjekkMyndighetsalder();
+        try {
+            var adresseBeskyttelse = personOppslagSystem.adresseBeskyttelse(request.annenPartFødselsnummer());
```

Comment:
må vi be om dokumentasjon når mor har beskyttet adresse?

Reply:
Jepp.  Han får ikke se mors plan i disse tilfellene.